nextflow.config

```diff
@@ -61,6 +62,11 @@ config_profile_contact = false
 config_profile_url = false
 }
 
+//export Time Zone required for QIIME2 2019.10
+env {
+  TZ = params.qiime_timezone
```

Comment:
Is there any way to just pass the host env vars through? If so, you could grab TZ, TIMEZONE, and any others that might be related.

Reply:
We tried doing this with the Singularity scope and passing through the environment variables such as TZ, but that didn't work as expected unfortunately :-(

Reply:
Not to state the obvious, but did you remember to do use the [singularity env var prefixing syntax](https://singularity.lbl.gov/docs-environment-metadata)? `SINGULARITYENV_TZ`

Reply:
We tried using the https://www.nextflow.io/docs/latest/config.html#config-singularity `envWhitelist` here, that should expose all set ENV parameters to the Singularity container, which didn't help in this case :-(